### PR TITLE
Removes unneeded select all checkbox in Survey List

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses #7513.  In consultation with @trahman73 it also introduces a new look for empty state that more closely follows patternfly's suggestion. 
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Screen Shot 2020-07-06 at 2 31 17 PM

### CODE REVIEW

File: awx/ui_next/src/screens/Template/Survey/SurveyList.jsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+  Title,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
 import styled from 'styled-components';
 import ContentLoading from '../../../components/ContentLoading';
-import ContentEmpty from '../../../components/ContentEmpty';
+// import ContentEmpty from '../../../components/ContentEmpty';
```
Comment (jlmitch5) [inline awx/ui_next/src/screens/Template/Survey/SurveyList.jsx]: can you remove this

**Review by jlmitch5 — APPROVED** [review]
looks good other than removing that commented out line

**Comment by unlikelyzero** [comment]
@AlexSCorey i'll test this one

**Comment by unlikelyzero** [comment]
I'm seeing both 
Screen Shot 2020-07-15 at 12 48 28 PM

**Comment by unlikelyzero** [comment]
Tests look good. No new e2e tests need to be added

Discovered this bug which is unrelated https://github.com/ansible/awx/issues/7636

**Pull request merged**